docs/IoT.js-API-UART.md

```diff
@@ -81,3 +85,27 @@ Closes UART device.
 * `callback: Function(err)`
 
 Writes data to UART device.
+
+
+## Particular information for Nuttx
+
+To use UART on stm32f4-discovery board, you must use proper pins. Stm32f4-discovery board supports 4 UART ports, such as USART2, USART3, UART5, USART6.
```

Comment:
As I know, there are USART1, USART4 on stm32f4-discovery board. Why don't you support them? Do you need to modify configure source file on nuttx?